client/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE HTML>
```

Comment:
Not sure - disregard if this is not the case - There are so many node_module files included here (github has only loaded the first 3,000 files), that I wondered if many of your package.json dependencies could be reclassified as dev dependencies - so that they would not be included in commits (and therefore the code reviews). I'm making a few assumptions which may not be true. 1) that dev dependencies are not included in the commits and code reviews, and 2)this bulky mass of files here is a result of dependencies.

Reply:
I don't see the package.json file or the server file. Maybe that will come in the next round of commits

Reply:
Hey Shain, it looks like your node_modules folder is excluded in your .gitignore file like it should be so I'm not sure what you were referring to with the first 3,000 files being loaded.

Reply:
Not sure why the package.json file isn't being included (it isn't gitignored), but I'll look into that. Changed dependencies to devDependencies.